config/samples/example_flags.json

```diff
@@ -67,6 +67,30 @@
           "off"
         ]
       }
+    },
+    "fibAlgo": {
+      "variants": {
+        "recursive": "recursive",
+        "memo": "memo",
+        "loop": "loop",
+        "binet": "binet"
+      },
+      "defaultVariant": "recursive",
+      "state": "ENABLED",
+      "targeting": {
+        "if": [
+          {
+            "$ref": "#/$evaluators/emailWithFaas"
```

Comment:
I think the way this is implemented might imply some functionality that doesn't exist.

`#/$evaluators/...` implies that some kind of pathing is supported, when it's really not.


I would recommend we just do something more like: 

```
"$evaluator": "emailWithFaas"
```

That's enough to imply it has some special treatment, but not that we look at some specified path, ie: `#/some/other/path`

Reply:
Thanks for the suggestion, done